tiledb/cf/engines/netcdf4_engine.py

```diff
@@ -60,10 +60,14 @@ class NetCDFCoordToDimConverter(SharedDim, NetCDFDimConverter):
 
     def __repr__(self):
         return (
-            f"Variable(name={self.input_name}, dtype={self.input_dtype}) -> "
+            f"NetCDFVariable(name={self.input_name}, dtype={self.input_dtype}) -> "
```

Comment:
Is this deliberately the same with the respective `NetCDFVariableConverter` repr?

Reply:
Yes, they map from the same thing (a NetCDF variable) to a new TileDB object that is defined by the super class.